src/components/Settings/Settings.js

```diff
@@ -67,12 +67,15 @@ const Settings = ( ) => {
 
   const queryClient = useQueryClient();
 
+  const invalidateQuery = useCallback( ( ) => {
+    queryClient.invalidateQueries( { queryKey: ["fetchUserMe"] } );
+  }, [queryClient] );
+
```

Comment:
Does this really force a re-fetch of users/me? If so can you please explain why in a comment in the code?

Reply:
My understanding is that anytime we invalidate queries, as long as we're making a fetch via useQuery, the fetch will automatically be refetched in the background. That's per this line in the [query invalidation docs](useUserMe) in React Query:

```
When a query is invalidated with invalidateQueries, two things happen:

It is marked as stale. This stale state overrides any staleTime configurations being used in useQuery or related hooks
If the query is currently being rendered via useQuery or related hooks, it will also be refetched in the background
```

This has been working well for me on other screens, but I'll verify that it's working here and add a comment.

Reply:
Is invalidating an existing query really what we want here? What if there is no existing query? The other issue here is that complicated logic surrounding how the `useUserMe` hook works should really be encapsulated in that hook, so I like the fact that it exported a `refetchUserMe` method. If invalidating an existing query is preferable to calling the `refetch` method returned by `useQuery`, I think we should change the implementation of that method rather than remove it and rely on other code to know how to invalidate queries.

Reply:
The reason I updated this code was because we're invalidating the query and then refetching, which I'm assuming means that React Query is running one refetch and then we're immediately repeating our own manual refetch. But I'm not attached to this, nor is it super relevant to this issue, so restoring for now.